release-notes/Cloud/dataminer_services_change_log_2025.md

```diff
@@ -9,6 +9,15 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 14 May 2025 - Fix - Invalid token after silent renew
+
+When a user session was silently renewed, the token could become invalid, causing the application to go into an invalid state that made it impossible to complete user actions. Tokens should now be correctly renewed and not block application actions.
+
+### 14 May 2025 - Fix - Admin - DxM updates giving false positives
+
+When an installed DxM version was different than the latest available version, the update message would be shown.
+The update message should now only be shown when the installed version is higher or equal to the latest available version.
```

Comment:
I'm not sure I fully understand this. What was the update message shown? If it's a message that an update is available, I wouldn't expect this to be displayed if the installed version is higher than or equal to the latest available version.